src/modules/expression/macros.rs

```diff
@@ -218,37 +218,6 @@ macro_rules! parse_expr {
     }};
 }
 
-#[macro_export]
-macro_rules! error_type_match {
```

Comment:
This macro could be removed entirely. I decided to remove unnecessary macros that introduce additional complexity to the compiler